docs/authentication.md

```diff
@@ -1,22 +1,30 @@
 ---
 sidebar_position: 30
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Rancher Harverster
+  - rancher harverster
```

Comment:
rancher harvester is no longer a keyword or project name, just harvester is enough.